src/decisionengine/framework/tests/WriteToDisk.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, config):
         print(f"WriteToDisk:{self.file.name}:")
         self.counter = 0
         self._consumes = {key: Any for key in config["consumes"]}
+        # TODO: need method to clean up file eventually
```

Comment:
It's probably more clear close to the open (line 49)

Reply:
Fixed.